charts/alfresco-repository/ci/kustomize/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+---
+namespace: alfresco
+secretGenerator:
+  - name: alfsecrets
+    literals:
+      - DATABASE_USERNAME=alfresco
+      - DATABASE_PASSWORD=alfresco
```

Comment:
this secret can be part of the chart, why force the user to provide a secret?

Reply:
At that stage there is no values to pass db credentials. Hence the required secret